### PR TITLE
Improved usability 

### PR DESCRIPTION
**Pull request by emieldesmidt**
This relatively small PR packs a significant overhaul of our voting demo application. 
The main list now displays cardView items, which provide a more intuitive way of handling the votes, whilst seeing more information. 

image

The purpose of this PR is to include this new layout in the main app, as it is a clear indication of where we are going.

### CODE REVIEW

File: voting/src/main/java/nl/tudelft/trustchain/voting/VotingActivity.kt [hunk 1]
```diff
@@ -137,6 +152,11 @@ class VotingActivity : AppCompatActivity() {
                 "proper JSON in its message field: ${block.transaction["message"]}."
         }
 
+        // Parse date field
+        val regex = Regex("^(.*?)GMT")
+        var strippedDate = regex.find(block.timestamp.toString())?.value.toString()
+        strippedDate = strippedDate.substring(0, strippedDate.length - 3)
```
Comment (MattSkala) [inline voting/src/main/java/nl/tudelft/trustchain/voting/VotingActivity.kt:158]: Consider using [DateFormat](https://developer.android.com/reference/java/text/DateFormat) to format dates, this looks very fragile

Comment (emieldesmidt) [reply voting/src/main/java/nl/tudelft/trustchain/voting/VotingActivity.kt]: This is definitely better, I've also implemented this change in the `blockListAdapter`.

**Comment by synctext** [comment]
is there an .APK build chain or push a fresh binary to the DAO branch?

screenshot: Dordrecht in fik? :laugh:

**Comment by devos50** [comment]
We might want to automate the .apk build process by integrating it in our [Jenkins server](https://jenkins-ci.tribler.org) (and build it on our BBQ server).

**Comment by MattSkala** [comment]
In #12, I have added a job to [Github Actions](https://github.com/Tribler/trustchain-superapp/actions) to build a debug APK on every push/pull request, so you can now get the latest APK for any branch in build artifacts.

I think it's easier to use than Jenkins and free for public repositories.

**Pull request merged**